packages/style-engine/README.md

```diff
@@ -131,13 +146,18 @@ Returns compiled CSS from a stored context, if found.
 _Parameters_
 
 -   _$store_name_ `string` An identifier describing the origin of the style object, e.g., 'block-supports' or ' global-styles'. Default is 'block-supports'.
+-   _$options_ `array<bool>` An array of options to determine the output.
+    -   _prettify_ `bool` Whether to add new lines and indents to output. Default is the test of whether the global constant `SCRIPT_DEBUG` is defined.
```

Comment:
would "Default is to inherit the value of `SCRIPT_DEBUG` if it is defined" be clearer, otherwise it seems like it could default to `true` if `SCRIPT_DEBUG` is defined by set to `false` ?

Reply:
Good pick up. Updated.